mm2src/crypto/Cargo.toml

```diff
@@ -60,3 +60,6 @@ tokio = { version = "1.20", default-features = false }
 
 [features]
 trezor-udp = ["trezor/trezor-udp"]
+
+[patch.crates-io]
+rand_core = { version = "0.6.2", package = "bip39" }
```

Comment:
Q: can't understand this patch. why doesn't it have a url/git? could you explain how/what this does.

Reply:
Yeah, I'm forcing bip39 to depend on rand_core 0.6.2 since the lib is selecting the least supported version(0.4.2) in kdf even tho the version isn't really compatible with the latest version of bip39
https://crates.io/crates/bip39/2.1.0/dependencies, I mean the feature of bip39 we using in kdf requires rand_core 0.6.2 or greater 

The lib is configured to select any version from 0.4.2==0.7